<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Терминал</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="SurfaceTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Terminal">
       <attribute name="title">
        <string>Терминал</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="OnIMSButton">
            <property name="text">
             <string>Включить IMS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="OffIMSButton">
            <property name="text">
             <string>Выключить IMS</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="labelCountSelection">
                <property name="text">
                 <string>Количество выборок:                   </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditCountSelection"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="labelTimeOpenGate">
                <property name="text">
                 <string>Период между выборками (мс):     </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditTimeOpenGate"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="labelDelayOpenGate">
                <property name="text">
                 <string>Длительность открытия затвора (мкс): </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDelayOpenGate"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="labelIonogramPeriod_2">
                <property name="text">
                 <string>Периодичность получения ионограмм (мс):</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditIonogramPeriod"/>
            </item>
            <item>
             <widget class="QLabel" name="labelSamplePeriod">
              <property name="text">
               <string>Задержка от открытия затвора до начала оцифровки (мкс):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSamplePeriod"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QPushButton" name="startButton">
                  <property name="text">
                   <string>Старт</string>
                  </property>
                  <property name="shortcut">
                   <string>Ctrl+G</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="stopButton">
                  <property name="text">
                   <string>Стоп</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="Timerlabel">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="NamePathFolderButton">
                  <property name="text">
                   <string>Выбрать папку для хранения данных:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="NamePathFolderlabel">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="LineBreakCheckBox">
                  <property name="text">
                   <string>Перенос строки</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="smoothiLineCheckBox">
                  <property name="text">
                   <string>Сглаживание углов на графике.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_18">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Отображение ионнограмм:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="CountIonSpinBox">
                    <property name="focusPolicy">
                     <enum>Qt::WheelFocus</enum>
                    </property>
                    <property name="frame">
                     <bool>true</bool>
                    </property>
                    <property name="readOnly">
                     <bool>false</bool>
                    </property>
                    <property name="accelerated">
                     <bool>false</bool>
                    </property>
                    <property name="correctionMode">
                     <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                    </property>
                    <property name="keyboardTracking">
                     <bool>true</bool>
                    </property>
                    <property name="showGroupSeparator" stdset="0">
                     <bool>false</bool>
                    </property>
                    <property name="suffix">
                     <string/>
                    </property>
                    <property name="minimum">
                     <number>10</number>
                    </property>
                    <property name="maximum">
                     <number>50000</number>
                    </property>
                    <property name="singleStep">
                     <number>10</number>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                    <property name="value">
                     <number>50</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Обновление 3D графика(с):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="settingConnButton">
                <property name="text">
                 <string>Настройки подключения</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+A</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="closeConButton">
                <property name="text">
                 <string>Разьединение</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+S</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="UpdateConnectComButton">
                <property name="text">
                 <string>Переподключиться</string>
                </property>
                <property name="shortcut">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Отправленые данные</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QScrollArea" name="scrollArea">
                   <property name="widgetResizable">
                    <bool>true</bool>
                   </property>
                   <widget class="QWidget" name="scrollAreaWidgetContents_3">
                    <property name="geometry">
                     <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>395</width>
                      <height>478</height>
                     </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_5">
                     <item row="0" column="0">
                      <widget class="QTextEdit" name="sendlabel">
                       <property name="readOnly">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Принятые данные</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QScrollArea" name="scrollArea_2">
                   <property name="widgetResizable">
                    <bool>true</bool>
                   </property>
                   <widget class="QWidget" name="scrollAreaWidgetContents_4">
                    <property name="geometry">
                     <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>395</width>
                      <height>478</height>
                     </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_6">
                     <item row="0" column="0">
                      <widget class="QTextEdit" name="labelExamSett">
                       <property name="readOnly">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="sendlineEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>отправка</string>
                </property>
                <property name="shortcut">
                 <string>Return</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="clearButtn">
              <property name="text">
               <string>Очистить окна</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Chart">
       <attribute name="title">
        <string>2D График</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="ChartVLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>3D график</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="SurfaceLayout"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="Buld3DChartButton">
          <property name="text">
           <string>Построить по последним данным в этом сеансе</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="GetDataForBuld3DChartButton">
          <property name="text">
           <string>Выбрать данные для построения </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
